ocs_ci/utility/utils.py

```diff
@@ -2558,6 +2558,25 @@ def ceph_health_resolve_crash():
     archive_ceph_crashes(ct_pod)
 
 
+def ceph_health_resolve_mon_slow_ops(health_status):
+    """
+    Fix ceph health issue with daemon crash
```

Comment:
Do we have daemon crash when slow ops is seen?

Reply:
copy paste issue, fixed